scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -33,8 +33,10 @@ package object dialects {
     allowXmlLiterals = true, // Not even deprecated yet, so we need to support xml literals
     toplevelSeparator = ""
   )
+    .withScalaVersion(2, 1, 0)
 
   implicit val Scala211 = Scala210
+    .withScalaVersion(2, 1, 1)
```

Comment:
```suggestion
    .withScalaVersion(2, 12, 0)
```
otherwise we risk people getting confused by this. Same in other places.